exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -277,7 +277,15 @@ def _translate_attributes(self, attributes) -> TypingSequence[KeyValue]:
                     logger.exception(error)
         return output
 
-    def _export(self, data: TypingSequence[SDKDataT]) -> ExportResultT:
+    def _export(
+        self,
+        data: TypingSequence[SDKDataT],
+        timeout_millis: Optional[float] = None,
+    ) -> ExportResultT:
+        if timeout_millis is not None:
+            timeout_seconds = timeout_millis / 10**3
+        else:
+            timeout_seconds = self._timeout
```

Comment:
Help me understand this. It is generally `self._timeout` except when we want to finish work as early as possible force_flush or shutdown in which case we pass the timeout_millis from processor/periodicexportingreader?